modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account/add_domains.jsp

```diff
@@ -38,6 +46,28 @@ String eventName = ParamUtil.getString(request, "eventName", liferayPortletRespo
 
 <aui:script>
 	function <portlet:namespace />addDomains() {
+		var domainsInput = document.getElementById('<portlet:namespace />domain');
+
+		var domains = domainsInput.value.split(',');
+
+		const pattern = /^([a-zA-Z0-9]+[a-zA-Z0-9-]*\.)+[a-zA-Z]{2,}$/;
```

Comment:
Double-check that there is not already a validator for this. If not, we should cite the source in the commit message.